build.gradle

```diff
@@ -33,7 +33,7 @@ buildscript {
 //
 // It must be false for fat aar generation of MuxExoPlayer to work
 // This is a bug/problem with the fat aar mechanism
-def inLocalDevModeFlag = false
+def inLocalDevModeFlag = true
```

Comment:
is this intended now?

Reply:
We should be able to undo this change once Mux Core v7.0.5 is released and referenced in the Gradle file